src/main/java/kr/pickple/back/crew/service/CrewMemberService.java

```diff
@@ -112,4 +112,25 @@ private Member findMemberById(final Long memberId) {
         return memberRepository.findById(memberId)
                 .orElseThrow(() -> new MemberException(MEMBER_NOT_FOUND, memberId));
     }
+
+    @Transactional
+    public void deleteMemberShip(final Long crewId, final Long memberId) {
+        final Crew crew = findByExistCrew(crewId);
+        final Member member = findMemberById(memberId);
+        //TODO:추후, 크루장인지 검증 로직 추가(11월 2일, 소재훈)
+
+        final Optional<CrewMember> crewMember = crewMemberRepository.findByMemberAndCrew(member, crew);
+
+        if (!crewMember.isPresent()) {
+            throw new CrewException(CREW_MEMBER_STATUS_NOT_FOUND, member.getId());
+        }
+
+        final CrewMember deleteCrewMember = crewMember.get();
```

Comment:
`p1`

1. Optinal로 받아서 isPresent()로 처리하는 것보다 orElseThrow를 이용하여 메서드 체이닝으로 처리하는 것이 가독성도 좋고 간결하게 처리할 수 있을 것 같습니다!
2. 현재 `CREW_MEMBER_STATUS_NOT_FOUND`라는 에러 명이 애매한 것 같습니다! 현재 해당 메서드는 CrewMember를 찾았을 때 없으면 예외이니 `CREW_MEMBER_NOT_FOUND`가 어떠실지 궁금합니다!

```suggestion
        final CrewMember crewMember = crewMemberRepository.findByMemberAndCrew(member, crew)
     .orElseThrow(() -> new CrewException(CREW_MEMBER_NOT_FOUND, crew.getId(), member.getId()));
```

Reply:
네~ 해당 부분 반영하여 수정하였습니다~
반영 커밋: